### PR TITLE
fix(cluster) simplify timer logic and ensure whole cluster detail pag…

### PR DESCRIPTION
**Pull request by edlerd**
…e refreshes, fix graph captions

### CODE REVIEW

File: ui/src/pages/clusters/metrics/ClusterTimer.tsx [hunk 1]
```diff
@@ -12,23 +9,12 @@ interface Props {
 const ClusterTimer: FC<Props> = ({ cluster }: Props) => {
   const [seconds, setSeconds] = useState(getSecondsSinceLastHeartbeat(cluster));
 
-  const { data: data } = useQuery({
```
Comment (mas-who) [inline ui/src/pages/clusters/metrics/ClusterTimer.tsx:15]: If we do the refetch at the `ClusterDetail` parent level, then all children components will re-render, is that okay?

Comment (edlerd) [reply ui/src/pages/clusters/metrics/ClusterTimer.tsx:15]: I think they should, because the graphs and stats will have changed.

Comment (mas-who) [reply ui/src/pages/clusters/metrics/ClusterTimer.tsx:15]: yeah that makes sense, LGTM then :+1:

**Review by mas-who — APPROVED** [review]

**Comment by mas-who** [comment]
@edlerd code and QA LGTM, thanks for the fixes :+1: Please would you have a look at #87 which fixes the CI errors. Once those are merged then we can rebase here.

**Pull request merged**